R/03_paths_rmarkdown.Rmd

```diff
@@ -0,0 +1,33 @@
+---
+title: "R Markdown Paths"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
```

Comment:
I would sort of expect to see `error = TRUE` here (and not `echo = TRUE`, which is a default), so that people can render this, even with broken paths, in order to see which types of paths work and which don't.